pontoon/base/views.py

```diff
@@ -430,7 +430,7 @@ def get_translation_history(request):
                 "uid": u.id,
                 "username": u.username,
                 "user_gravatar_url_small": u.gravatar_url(88),
-                "user_status": u.status(locale),
+                "user_status": u.status(locale, t.entity.resource.project),
```

Comment:
This makes 3 additional DB queries for each translation loaded in the history panel.